praw/models/listing/mixins/submission.py

```diff
@@ -13,6 +13,17 @@ def duplicates(self, **generator_kwargs):
         Additional keyword arguments are passed in the initialization of
         :class:`.ListingGenerator`.
 
+        Example usage:
+
+        .. code:: python
+
+            submission = reddit.submission(id='5or86n')
```

Comment:
For consistency, these lines should be indented as far as `code` above. I think as-is it's one space too far.